dask_cudf/tests/test_join.py

```diff
@@ -220,7 +220,12 @@ def test_indexed_join(how):
 
     d = g_left.merge(g_right, left_index=True, right_index=True, how=how)
     dg = dg_left.merge(dg_right, left_index=True, right_index=True, how=how)
-    dd.assert_eq(d, dg)
+
+    # occassionally order is not correct (possibly do to hashing in the merge)
+    d = d.sort_values('x')  # index is preserved
+    dg = dg.sort_values('x')  # index is reset -- sort_values will slow test down
```

Comment:
Is this entirely intermittent, or is it version dependent

("I don't know" is a perfectly fine answer)

Reply:
I don't know -- @raydouglass noticed that the tests didn't pass on different hardware:
>  P100 vs V100